cfpb/cfgov-refresh
# Update Ask CFPB category page navigation

Add environment check before importing ask_cfpb models

## Additions

- Add environment check before importing Ask Category model in `instanceOfBrowseOrFilterablePages`

## Changes

- Use default sibling-page secondary nav generation for category pages

## Testing

- Check out branch 
- Set up Ask if necessary: 
  ```
  export DEPLOY_ENVIRONMENT=build
  ./drop-db.sh
  ./create-mysql-db.sh
  mysql -uroot v1 < ~/Downloads/production_django.sql
  python cfgov/manage.py migrate
  python cfgov/manage.py runscript migrate_knowledgebase
  ```
- Verify that secondary navigation is still correctly generated on [Ask category](http://localhost:8000/ask-cfpb/category-bank-accounts-and-services/) and existing [browse](http://localhost:8000/about-us/the-bureau) pages

## Screenshots

<img width="1312" alt="screen shot 2017-04-19 at 4 32 43 pm" src="https://cloud.githubusercontent.com/assets/778171/25206599/e0874da6-251d-11e7-83e4-f432368862ff.png">



## Checklist
